calyx/src/lang/context.rs

```diff
@@ -210,6 +224,8 @@ impl Into<ast::NamespaceDef> for Context {
         ast::NamespaceDef {
             name: name.into(),
             components,
+            ///TODO: fix it later
```

Comment:
TODO comments should make sense to other collaborators! Please follow the [Google Styleguide](https://google.github.io/styleguide/cppguide.html#TODO_Comments) for this.